lib/tower_cli/resources/job_template.py

```diff
@@ -76,7 +75,7 @@ def create(self, extra_vars=None, *args, **kwargs):
         return super(Resource, self).create(*args, **kwargs)
 
     @resources.command
-    def modify(self, *args, **kwargs):
+    def modify(self, pk=None, *args, **kwargs):
```

Comment:
Note: arguments need to be declared like this in order for the click library to accept command line use of the command, like "command 15", and if not, that usage will throw an error. I thought this type of detail might be important to go in a readme for contributing at some point.